source/test/F0.Minesweeper.Components.Tests/Logic/Cell/CellStatusTranslationTests.cs

```diff
@@ -38,7 +35,8 @@ public void GetDisplayValue_NoActiveTranslationSet_NoAdjacentMines_Throws()
 
 			// Act && Assert
 			Action actionToTest = () => instanceUnderTest.GetDisplayValue(null);
-			actionToTest.Should().Throw<ArgumentNullException>();
+			actionToTest.Should().ThrowExactly<ArgumentNullException>()
+				.And.ParamName.Should().Be("adjacentMineCount");
```

Comment:
In case of a renaming of the param the test won't pick this up unless someone manually changes the test. Is this desired by us? 
From my side it's not a blocker. Just something that I personally would usually avoid since this doesn't really test functionality.

Reply:
Since the `GetDisplayValue` is internal I guess it's fine to ignore the `ParamName`.

Reply:
Discussed: We will remove the assertion of `ParamName` again.